index.js

```diff
@@ -759,3 +761,22 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.not = negateMethods(is);
+
+function negateMethods(source) {
+  var negatedMethods = {};
+  forEach(source, function (value, key) {
+    var negatedMethod;
+    if (value instanceof Function) {
```

Comment:
`instanceof` is unreliable - ideally we should use the `is-callable` module here, or if not, `typeof value === 'function'` which would only report false positives on regexes on older engines (`is-callable` handles that case)